crates/sui-protocol-config-macros/src/lib.rs

```diff
@@ -52,12 +75,33 @@ pub fn getters_macro(input: TokenStream) -> TokenStream {
                         } else {
                             panic!("Expected angle bracketed arguments.");
                         };
-                        Some(quote! {
+
+                        let getter =quote! {
```

Comment:
nit spacing

Reply:
Fixed in this PR https://github.com/MystenLabs/sui/pull/11767